### PR TITLE
FPK-277: Edit README.md with correct format

### PR DESCRIPTION
**Pull request by donovanroubos**
The README.md didn't had a proper format to meet the requirements. Now all README.md will consist of formatted templates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +16,14 @@ npm install react-flagpack --save
...
+### Defining the component
+Import the Flagpack component into any of your project's React components.
 ```js
-// use own name as variable
 import Flag from 'react-flagpack'
 ```
 
-Use in JSX:
+### Use in JSX
 ```html
```
Comment (daoneandonly) [inline README.md:27]:
```suggestion
```jsx
```
We can use jsx syntax here. It's a minor change, but this is technically not html. It'll show up as follows:

``````jsx
<Flag
  code="NL"
  hasDropShadow
/>

File: README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
-# React Flagpack
+# Flagpack
```
Comment (daoneandonly) [inline README.md:1]: We should consider use [Shields](https://shields.io) at the top of the project `readme.md`. It's used in a lot of Open Source projects and is somewhat the standard. It gives the user/contributer a clear overview of the current status of the project, but these icons can be set up pretty dynamically.

File: README.md [hunk 3]
```diff
@@ -62,3 +73,21 @@ npm publish
...
+
+You can find a changelog of Flagpack's releases on the [Releases page](https://github.com/Yummygum/react-flagpack/releases) on GitHub.
+
+## Contribute
+
+If you're interested in contributing to this project, great! Please see the [contributing document](CONTRIBUTING.md).
+
+## License
+
+Flagpack is an open source project published under a [MIT license](LICENSE).
```
Comment (daoneandonly) [inline README.md:93]: I feel like we can give a little shoutout to our [Yummygum website](https://www.yummygum.com/) here at the end. No harm in linking to show who is behind this project ❤️.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,13 @@
...
+Flagpack contains 260+ flag icons to easily use within your code project. Flagpack is an open source project and available for JavaScript frameworks/libraries Angular, Vue and React.
+
+![Flagpack. 260+ easily implementable flag icons to use in your design or code project. Open Source. Available for Sketch, Figma, Angular, Vue, and React. [www.flagpack.xyz](http://www.flagpack.xyz). Made with love by Yummygum. Graphic showing a list with the flags of Argentina, Croatia, Estonia, Kenia, Netherlands, and Scotland.](https://flagpack.xyz/meta-image.png)
+
+[View documentation on flagpack.xyz](https://flagpack.xyz/docs/)
+
+## react-flagpack
+Flagpack for React is created using React v16.8. The Flagpack component has not been tested for compatibility with older version of React.
 
 ## Installation
```
Comment (daoneandonly) [inline README.md:12]: It's a bit nitpicky, but should we set an example to run the project with `yarn` aswell? Yarn is apparently [pretty close in usage](https://www.npmtrends.com/npm-vs-yarn) compared to npm.

**Review by daoneandonly — CHANGES_REQUESTED** [review]
I have some points I'd like you to have a look at.

File: README.md [hunk 5]
```diff
@@ -62,3 +73,21 @@ npm publish
 ```
 
 When you're confident with the release, make sure the version tag is also released at GitHub.
+
+## Support
+
+[Frequently Asked Questions](http://dev.flagpack.xyz/support/)
+
+[Documentation](http://dev.flagpack.xyz/docs/)
```
[the remark this answers is not in the document]
Comment (donovanroubos) [reply README.md:81]: Good catch

File: README.md [hunk 6]
```diff
@@ -23,28 +32,30 @@ Use in JSX:
...
 ```bash
 # install dependencies
 npm install
 
 # build for prod
 npm run build
+
+# build with watcher for dev
+npm run dev
 ```
```
[the remark this answers is not in the document]
Comment (donovanroubos) [reply README.md:59]: I think, people who use yarn also know how to install. Maybe we'll add this later.

File: README.md (hunk 2 above)
Comment (donovanroubos) [reply README.md:1]: Like it! I'll add it to Jira as new task.

**Review by daoneandonly — APPROVED** [review]

**Pull request merged**